src/transports/smart_protocol.c

```diff
@@ -19,6 +19,8 @@
 /* The minimal interval between progress updates (in seconds). */
 #define MIN_PROGRESS_UPDATE_INTERVAL 0.5
 
+bool git_ofs_delta__enabled = true;
```

Comment:
This name doesn't quite make sense in our naming scheme -- we try to scope things to something sort of like an "object" or - when that stops making sense - the file level.  (I concede that given the filename, `smart_protocol.c`, this has stopped making sense here, since this was logical outgrowth from `smart.c` which I guess just got too big).  So `git_smart` is the object, or concept here, and _it's_ the prefix.  Anything that applies to that scope should be double underscored.

Something like:  `git_smart__ofs_delta_enabled`.

(If you squint really hard it seems like we're trying to make C look object oriented, which is either terrible or merely odd depending on your mood.)